src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Setup/SetupComponentRegistrationService.cs

```diff
@@ -266,4 +271,47 @@ public int OnAfterCloseSolution(object pUnkReserved)
     }
 
     #endregion
+
+    #region Runtime component ID handling
+
+    private static string? TryGetNetCoreRuntimeComponentId(string versionString)
```

Comment:
Could we add a test for this method?

Reply:
Good call. Added tests and found a bug...